framework/src/st_things/things_stack/port_tinyara.c

```diff
@@ -97,3 +97,9 @@ unsigned int if_nametoindex(const char *ifname)
 {
 	return 0;					// TODO: Now supports only 1 device
 }
+
+const char *gai_strerror(int errcode)
+{
+	static const char *n_str = "null";
```

Comment:
how about use string  that have more clear meaning here instead of null?

Reply:
Fixed, thanks.